plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -130,7 +138,7 @@ func TestPodFitsResources(t *testing.T) {
 	}
 
 	for _, test := range enoughPodsTests {
-		node := api.Node{Status: api.NodeStatus{Capacity: makeResources(10, 20, 32).Capacity}}
+		node := api.Node{Status: api.NodeStatus{Capacity: makeResources(10, 20, 32).Capacity, Allocatable: makeAllocatableResources(10, 20, 32)}}
```

Comment:
@timothysc are you asking about the test or about the feature? API is here
https://github.com/kubernetes/kubernetes/pull/18189/files

IIUC the scheduler only needs to work with allocatable because we are guaranteed to either be using Kuebelets that publish allocatable, or the API server will default allocatable from capacity if Kubelet is not publishing allocatable.